### PR TITLE
Tutorial improvements.

### PR DESCRIPTION
**Pull request by statwonk**
Various suggested improvements to the tutorial for clarity.

### CODE REVIEW

**Comment by mistercrunch** [comment]
"#  variables t1, t2 and t3 are examples of tasks created by instantiating operators"

Also we should keep the file `airflow/example_dags/tutorial.py` in sync with this `rst` file

**Comment by statwonk** [comment]
Thanks. :) The term constructor threw me for a loop, any ideas for being more explicit about that?

File: docs/tutorial.rst [hunk 1]
```diff
@@ -123,8 +124,9 @@ We also pass the default argument dictionary that we just define.
 
 Tasks
 -----
-Tasks are generated when instantiating objects from operators. The first
-argument ``task_id`` acts as a unique identifier for the task.
+Tasks are generated when instantiating objects from operators. An object
+instatiated from an operator is called a constructor. The first argument
```
Comment (mistercrunch) [inline docs/tutorial.rst]: The constructor is the name of the special method that define how the object is created. It's an OOP term, not specific to Airflow or anything.

Comment (statwonk) [inline docs/tutorial.rst]: Got it, apologies, I'm coming from a FP background (R). :)

**Comment by mistercrunch** [comment]
Apart from the confusion around constructor everything looks much better than it did before, thanks for taking the time.

nit: this whole PR probably should be a single commit (or 2 since there's comment history), note that you can `--amend` between pushes

**Comment by statwonk** [comment]
Glad you think so! I'd really like to use AirFlow in production --- it fulfills an immediate need, so I'd like to contribute where I can. 10-4 on the verbose commits. I'll be sure to squash them before I submit next time.

**Comment by mistercrunch** [comment]
![img] (http://media.giphy.com/media/IB9foBA4PVkKA/giphy.gif)

**Pull request merged**